redash/settings.py

```diff
@@ -150,7 +150,7 @@ def all_settings():
 # LDAP Search DN TEMPLATE (for AD this should be "(sAMAccountName=%(username)s)"")
 LDAP_SEARCH_TEMPLATE = os.environ.get('REDASH_LDAP_SEARCH_TEMPLATE', '(cn=%(username)s)')
 # The schema to bind to (ex. cn=users,dc=ORG,dc=local)
-LDAP_SEARCH_DN = os.environ.get('REDASH_SEARCH_DN', None)
+LDAP_SEARCH_DN = os.environ.get('REDASH_LDAP_SEARCH_DN', None)
```

Comment:
We need to support both, just in case someone already using REDASH_SEARCH_DN, so something like this:
`os.environ.get('REDASH_LDAP_SEARCH_DN', os.environ.get('REDASH_SEARCH_DN'))`.

Reply:
Thanks for the review, I will update the PR.